decidim-core/db/seeds.rb

```diff
@@ -102,6 +102,7 @@
     tos_agreement: true,
     personal_url: Faker::Internet.url,
     about: Faker::Lorem.paragraph(2)
+    # accepted_tos_version: Time.current
```

Comment:
🧐

Reply:
I'll fix it :sweat: this sould be `accepted_tos_version: organization.tos_version`